internal/core/converter/tensorflow/tensorflow.go

```diff
@@ -22,59 +22,73 @@ import (
 	"github.com/vdaas/vald/internal/errors"
 )
 
+// SessionOptions is a type alias for tensorflow.SessionOptions.
 type SessionOptions = tf.SessionOptions
+
+// Operation is a type alias for tensorflow.Operation.
 type Operation = tf.Operation
 
+// TF represents a tensorflow interface.
 type TF interface {
 	GetVector(inputs ...string) ([]float64, error)
 	GetValue(inputs ...string) (interface{}, error)
 	GetValues(inputs ...string) (values []interface{}, err error)
+	Closer
+}
+
+type session interface {
+	Run(feeds map[tf.Output]*tf.Tensor, fetches []tf.Output, operations []*Operation) ([]*tf.Tensor, error)
+	Closer
+}
+
+// Closer close a tensorflow.Session.
+type Closer interface {
```

Comment:
sorry, could you please delete `Closer` interface?
And Could you please write type definition?
eg)
type Closer io.Closer